docs/runners/local.md

```diff
@@ -14,31 +14,98 @@ In order to use the local runner, you need to have a recent version of [docker-c
 Make sure that you have installed Docker compose on your system. You can find more information 
 about this in the [installation](../guides/installation.md) guide.
 
-```bash
-#### Running a Docker compiled pipeline
 
-```bash
-fondant run local <pipeline_ref>
-```
-
-NOTE: that the pipeline ref is the path to the compiled pipeline spec OR a reference to a fondant pipeline in which case a Docker compiler will compile the pipeline
-to a docker compose specification before running the pipeline.This will start the pipeline and provide logs per component (service).
+=== "Console"
+
+    ```bash
+    fondant run local <pipeline_ref>
+    ```
+
+    If you want to use remote paths (GCS, S3, etc.) you can use the `--auth-gcp`, `--auth-aws` or `--auth-azure`.
```

Comment:
Maybe it makes sense to split it up as well (similar to the example below). 
Selecting/activate the right session/profile for different cloud providers works differently. 
If we would add the commands for each cloud provider everything would be here on the page.